aws-event-sources-operator/build/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM quay.io/operator-framework/helm-operator:v0.18.2
+
+LABEL name="AWS Event Sources Operator" \
+      vendor="Triggermesh Inc." \
+      version="v0.0.1" \
+      release="1" \
+      summary="AWS Event Sources controller for Kubernetes" \
+      description="aws-event-sources is a Kubernetes controller that implements Knative event sources for AWS services"
```

Comment:
Didn't you mention multiple LABEL directives were squashed since Docker 1.10, making it unnecessary to use those escapes?

Reply:
yep. copy pasted the snippet from redhat docs. will split